characters_counter/characters_counter.py

```diff
@@ -1,16 +1,56 @@
-from collections import Counter
+import collections
+import re
+from collections import Counter, defaultdict
 from functools import cache
+from pathlib import Path
 
 REQUIREMENT_OCCURRENCE = 1
+NUM_OF_MOST_COMMON_CHARACTERS = 3
 
 
 @cache
 def count_characters_occurrence(string: str) -> int:
     """
-    The function returns the number of characters in the string occuring only
-    once.
+    The function returns the number of characters in the string occuring only once.
     """
     characters_counts = Counter(string)
     unique_occurrences = filter(lambda occurrence: (occurrence == REQUIREMENT_OCCURRENCE), characters_counts.values())
     unique_characters_number = sum(unique_occurrences)
     return unique_characters_number
+
+
+def get_frequent_characters_per_sentence(sentence: str) -> dict:
```

Comment:
dict -> dict[key, value]
Please be more specific with typing

Reply:
Ok, the mypy checks were passed and I thought that it was enough